src/clients/sidecar/models/ScaffoldV1TemplateOption.ts

```diff
@@ -0,0 +1,144 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Confluent ide-sidecar API
+ * API for the Confluent ide-sidecar, part of Confluent for VS Code
+ *
+ * The version of the OpenAPI document: 1.0.1
+ * Contact: vscode@confluent.io
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { mapValues } from "../runtime";
+/**
+ *
+ * @export
+ * @interface ScaffoldV1TemplateOption
+ */
+export interface ScaffoldV1TemplateOption {
+  /**
+   *
+   * @type {string}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  display_name: string;
+  /**
+   *
+   * @type {string}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  description: string;
+  /**
+   *
+   * @type {string}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  hint?: string;
+  /**
+   *
+   * @type {string}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  format?: string;
+  /**
+   *
+   * @type {string}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  pattern?: string;
+  /**
+   *
+   * @type {Array<string>}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  _enum?: Array<string>;
+  /**
+   *
+   * @type {string}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  initial_value?: string;
+  /**
+   *
+   * @type {number}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  min_length?: number;
+  /**
+   *
+   * @type {Array<string>}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  examples?: Array<string>;
+  /**
+   *
+   * @type {Array<string>}
+   * @memberof ScaffoldV1TemplateOption
+   */
+  enum?: Array<string>;
```

Comment:
I had to fix this from `_enum` (duplicated) -> `enum` manually. Which means the next time we run `npx gulp apigen` my changes will be lost. We'll need to file a bug report with openapi-generator to figure out what's going on here.